src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -70,8 +112,30 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
   public void init() throws ServletException {
     super.init();
     setUserStore(UserStore.getInstance());
+    //adding associations
+    registerTest.put(0, rPic0Address );
+    registerTest.put(1, rPic1Address);
+    registerTest.put(2, rPic2Address);
+    registerTest.put(3, rPic3Address);
+    registerTest.put(4, rPic4Address);
+    registerTest.put(5, rPic5Address);
+    registerTest.put(6, rPic6Address);
+    registerTest.put(7, rPic7Address);
+    registerTest.put(8, rPic8Address);
+    registerTest.put(9, rPic9Address);
+    answers[0] = rPic0;
```

Comment:
You can shorten this by doing `answers = {rPic0, rPic1, ...};`

Reply:
Hey thanks for the tip. The compiler is saying array constants can only be used in initializers when I tried doing that shortening. Since it's so close to the deadline I may just keep it in this form, although stylistically it isn't ideal.